grails-app/domain/org/pih/warehouse/order/OrderAdjustment.groovy

```diff
@@ -58,6 +58,17 @@ class OrderAdjustment implements Serializable {
         return amount ?: percentage ? orderItem ? orderItem?.subtotal * (percentage/100) : order.subtotal * (percentage/100) : 0
     }
 
+    def getInvoiceItem() {
```

Comment:
Couldn't there be multiple invoice items for a single order adjustment?

Reply:
I think that technically it can have only one invoice item. Because it has quantity = 1. And there won't be invoice item candidate if order adjustment already have invoice item associated.
https://github.com/openboxes/openboxes/blob/develop/grails-app/migrations/views/invoice-candidate-view.sql#L26